<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CenterView</class>
 <widget class="QWidget" name="CenterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabView">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="interEditor">
      <attribute name="title">
       <string>面试设置</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="issueEditor">
      <attribute name="title">
       <string>考题设置</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="stuEditor">
      <attribute name="title">
       <string>考生录入</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="interManeger">
      <attribute name="title">
       <string>面试管理</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
